xwiki-commons-core/xwiki-commons-cache/xwiki-commons-cache-infinispan/src/test/java/org/xwiki/cache/infinispan/InfinispanConfigTest.java

```diff
@@ -115,7 +115,7 @@ void createCacheWhenFileCacheNoPath() throws Exception
         assertEquals(1, stores.size());
 
         String location = ((SingleFileStoreConfiguration) stores.get(0)).location();
-        assertEquals(this.environment.getTemporaryDirectory().toString() + "/cache", location);
+        assertEquals(this.environment.getTemporaryDirectory().toString() + "\\cache", location);
```

Comment:
You seem to have converted a test that was based on UNIX separator ("/") to Windows separator ("\"), this means it's going to fail everywhere else than Windows.

Reply:
I will revert this change. Thanks for noticing.